### ISSUE TITLE
Add DeviceID Reset

### ISSUE THREAD

**Issue by Long-Man** [Montellese/mediaimporter.emby#39]
Owing to long sync times on low powered devices, it is useful to be able to sync on a more powerful device and copy the database. It is necessary to generate a new DeviceID on the new device.

**Comment by Montellese** [comment]
Can you give the [feature/reset_deviceid](https://github.com/Montellese/mediaimporter.emby/compare/feature/reset_deviceid) a try? It adds an action setting to the Emby media provider settings which generates a new device ID.

Since this also affects the observer and player threads running in the background Kodi has to be restarted after generating a new device ID and saving the media provider's settings. There's a dialog popping up with these instructions as well.

**Comment by Long-Man** [comment]
I can confirm that it works as expected. I copied the database from an intel system unto a Pi4. When I restarted the Pi4, it show in the server with the name of the intel system. I then genertated a new DeviceID and restarted. Now It shows with the correct name and a fully synced database.

Thanks for this. working with the Pi has been a pain.

I also tested the no password option and can confirm that it works also.
